### PR TITLE
Update README to reflect recent additions

### PR DESCRIPTION
**Pull request by guillaumekln**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ The project is production-oriented and comes with [backward compatibility guaran
...
+* **Fast and efficient execution on CPU and GPU**<br/>The execution [is significantly faster and requires less resources](#benchmarks) than general-purpose deep learning frameworks on supported models and tasks.
 * **Quantization and reduced precision**<br/>The model serialization and computation support weights with reduced precision: 16-bit floating points (FP16), 16-bit integers, and 8-bit integers.
+* **Multiple CPU architectures support**<br/>The project supports x86-64 and ARM64 processors and integrates multiple backends that are optimized for these platforms: [Intel MKL](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html), [oneDNN](https://github.com/oneapi-src/oneDNN), [OpenBLAS](https://www.openblas.net/), and [Apple Accelerate](https://developer.apple.com/documentation/accelerate).
+* **Automatic CPU detection and code dispatch**<br/>One binary can include multiple backends (e.g. Intel MKL and oneDNN) and instruction set architectures (e.g. AXV, AVX2) that are automatically selected at runtime based on the CPU information.
```
Comment (keichi) [inline README.md:27]: Spotted a typo: AXV -> AVX

File: README.md [hunk 2]
```diff
@@ -21,11 +21,12 @@ The project is production-oriented and comes with [backward compatibility guaran
...
 * **Parallel translations**<br/>CPU translations can be run efficiently in parallel without duplicating the model data in memory.
 * **Dynamic memory usage**<br/>The memory usage changes dynamically depending on the request size while still meeting performance requirements thanks to caching allocators on both CPU and GPU.
-* **Automatic CPU detection and code dispatch**<br/>The fastest code path is selected at runtime based on the CPU (Intel or AMD) and the supported instruction set architectures (AVX, AVX2, or AVX512).
 * **Ligthweight on disk**<br/>Models can be quantized below 100MB with minimal accuracy loss. A full featured Docker image supporting GPU and CPU requires less than 400MB.
```
Comment (keichi) [inline README.md:30]: Another one: Ligthweight -> Lightweight

File: README.md (hunk 1 above)
Comment (guillaumekln) [reply README.md]: Thanks for the review!

File: README.md (hunk 2 above)
Comment (guillaumekln) [reply README.md]: Thanks! :)

**Pull request merged**